dev-tools/releng/src/main.rs

```diff
@@ -428,6 +428,42 @@ async fn main() -> Result<()> {
             .env_remove("RUSTUP_TOOLCHAIN"),
     );
 
+    // Record the commit of helios.git and everything helios-setup cloned
+    {
+        let git_bin = args.git_bin.clone();
+        let output_dir = args.output_dir.clone();
+        let helios_dir = args.helios_dir.clone();
+        let logger = logger.clone();
+        jobs.push("helios-record", async move {
+            let projects_dir = helios_dir.join("projects");
+            let mut projects = BTreeMap::new();
+            let mut read_dir = fs::read_dir(&projects_dir).await?;
```

Comment:
Could use `projects_dir.read_dir_utf8` here -- it isn't async but ehh, I don't think it matters.